src/DotLiquid.Tests/ContextTests.cs

```diff
@@ -785,34 +790,78 @@ public void TestNestedDynamicAsVariable()
             Assert.AreEqual("Hello", _context["dynamic.lambda.name"]);
         }
 
+        private class ExpandoModel

```

Comment:
Consider grouping related privates and test methods in a separate class to avoid this test class from growing very large and difficult to maintain.

Reply:
Removed one unused private class and relocated the other to test class region.